core/jobs_registry.py

```diff
@@ -97,6 +94,17 @@
 # List of all manager classes for prod validation one-off batch jobs for which
 # to show controls on the admin dashboard.
 AUDIT_JOB_MANAGERS = [
+    (
+        interaction_validation_jobs_one_off
+        .DragAndDropSortInputInteractionOneOffJob),
+    (
+        interaction_validation_jobs_one_off
+        .InteractionRTECustomizationArgsValidationOneOffJob),
+    interaction_validation_jobs_one_off.ItemSelectionInteractionOneOffJob,
+    interaction_validation_jobs_one_off.MultipleChoiceInteractionOneOffJob,
+    (
+        interaction_validation_jobs_one_off
+        .InteractionCustomizationArgsValidationOneOffJob),
```

Comment:
Add this in sorted order* i.e, before line 103?

Reply:
Updated.